### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by frappacchio** [johngodley/redirection#648]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 5.6.32
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Nessun percorso fornisce una corrispondenza tra l'URL e le modalità di richiesta
Action: redirection/v1/bulk/group/disable/?per_page=50 POST
Params: {"items":"3"}
Code: 404 
Raw: {"code":"rest_no_route","message":"Nessun percorso fornisce una corrispondenza tra l'URL e le modalit\u00e0 di richiesta","data":{"status":404}}
```

**Comment by frappacchio** [comment]
Solved by downgrading to 2.10.1

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
